typedoc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "entryPoints": [
+        "src/extension.ts",
+        "src/core/commandRunner/CommandRunner.ts",
+        "src/core/inferFullTargets.ts",
+        "src/core/updateSelections/RangeUpdater.ts",
+        "src/core/updateSelections/updateRangeInfos.ts",
+        "src/core/updateSelections/updateSelections.ts",
+        "src/processTargets/index.ts",
+        "src/processTargets/modifiers/processModifier.ts",
+        "src/processTargets/processMark.ts",
+        "src/typings/Types.ts",
+        "src/typings/treeSitter.ts",
+        "src/util/canonicalizeAndValidateCommand.ts",
+        "src/util/getPrimitiveTargets.ts"
```

Comment:
how did you choose these entrypoints?  Why not just everything?

Reply:
Thats is what I will do now. For some reason I thought to only create docs for 'important modules'. And yes they were hand picked by me